form.yml.erb

```
@@ -22,10 +22,14 @@ form:
   - classroom
 attributes:
   jupyterlab_switch:
-    widget: "check_box"
-    label: "Use JupyterLab instead of Jupyter Notebook?"
+    widget: "radio"
+    label: ""
+    value: "1"
     help: |
       JupyterLab is the next generation of Jupyter, and is completely compatible with existing Jupyter Notebooks.
+    options:
+      - ["1","Jupyter Lab"]
+      - ["0", "Jupyter Notebook"]
```

Does this work? These 2 changes.

```suggestion
      - ["lab","Jupyter Lab"]
      - ["notebook", "Jupyter Notebook"]
```
This change in `template/script.sh.erb`.  
jupyter <%= context.jupyterlab_switch %> --config="${CONFIG_FILE}"

I'm wondering if we can pass strings directly - why cast them to integers?